core/domain/story_services_test.py

```diff
@@ -1663,18 +1691,23 @@ def test_migrate_story_contents_to_latest_schema(self):
             description='A new topic', canonical_story_ids=[],
             additional_story_ids=[], uncategorized_skill_ids=[],
             subtopics=[], next_subtopic_id=0)
-        self.save_new_story(
-            story_id, user_id, 'Title', 'Description', 'Notes',
-            topic_id)
-
-        story_model = story_models.StoryModel.get(story_id)
-        self.assertEqual(story_model.story_contents_schema_version, 1)
+        story_model = story_models.StoryModel(
+            id=story_id,
+            description='Description',
+            title='Title',
+            language_code='1',
+            story_contents_schema_version=1,
+            notes='Notes',
+            corresponding_topic_id=topic_id,
+            story_contents=self.VERSION_1_STORY_CONTENTS_DICT
+        )
 
-        swap_story_object = self.swap(story_domain, 'Story', MockStoryObject)
         current_schema_version_swap = self.swap(
```

Comment:
This doesn't seem right. It should be v1 first and then the 'get' function should migrate it to 2.

Reply:
I just followed #9095 to implement it here. I am not sure as to why it is even required because CURRENT_STORY_CONTENTS_SCHEMA_VERSION is already 2. How should I implement it?

Reply:
Hi @rohitkatlaa, just a general comment, please be sure about code you commit. If you have doubts, you can comment on the PR you are following. Do not copy-paste stuff if you do not understand why it is required.

Reply:
@kevintab95 I am extremely sorry. I didn't realize it until @aks681 pointed it out.

Reply:
Thanks! Regarding your question, I think the pattern we follow in such tests is to check if the migration works correctly for the next version ie. swap 2 with 3 because the current version is 2 and the migrated version should be 3. For reference, this seems to be the approach [here](https://github.com/oppia/oppia/blob/develop/core/domain/skill_services_test.py#L1113). @aks681 did I get that right? Please advise.

Reply:
Oh, actually nvm, the supposed is supposed to be there, so need to remove the swap line.
The reason being that in case we migrate it to other versions, for this particular test, we need the target version to be v2, so the swap needs to be there to fix the destination.
Also, @kevintab95, for further migrations, we would be creating new functions (I think), so no need to edit this function if we migrate to a v3 later on. So, we need this swap line.

Reply:
Ah, I see. Makes sense. Thanks for the explanation @aks681!

Reply:
Ok I too get it now. Thanks for the explanation @aks681 .